src/web/routes/entities/integration.test.ts

```diff
@@ -447,6 +447,136 @@ describe('/entities', () => {
 				},
 			]);
 		});
+		it('should handle when the last touched by is not available', async () => {
```

Comment:
**suggestion (non-blocking):** This seems like use case, which can be cheaper and easier to cover with unit tests, as it's related to a special case and its handling is fully encapsulated in `transformers`. 

Please, consider, adding unit tests to `transformers` instead.